src/Game.js

```diff
@@ -0,0 +1,80 @@
+class Game {
+	constructor(frame = new Frame()) {
+		this.frameNo = 1;
+		this.frames = [];
+		this.currentFrame = frame;
+		this.currentscore = 0;
+		this.gameOver = false;
+	}
+
+	nextFrame(frame = new Frame()) {
```

Comment:
This whole method could be much shorter:
```js
nextFrame() {
  if (this.frames.length < 9) {
    this.frames.push(new this.frameClass())
  } else if (this.frames.length === 9) {
    this.frames.push(new this.finalFrameClass())
  }
}
```

Something like that. I've kept the current frame within the frame array here - not sure why you are keeping it separate.